<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MALAPI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_search">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <widget class="QTableWidget" name="listSearch">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>60</y>
          <width>776</width>
          <height>491</height>
         </rect>
        </property>
        <property name="columnCount">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSearch">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>10</y>
          <width>111</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="textSearch">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>361</width>
          <height>40</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_list">
       <attribute name="title">
        <string>List</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="listList"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionAdd_to_list">
   <property name="text">
    <string>Add to list</string>
   </property>
  </action>
  <action name="actionDelete_from_list">
   <property name="text">
    <string>Delete from list</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
